language-support/hs/bindings/stack.yaml

```diff
@@ -0,0 +1,18 @@
+resolver: lts-14.3
+
+extra-lib-dirs:
+- /usr/local/grpc/lib
+
+extra-deps:
+
+- git: https://github.com/awakesecurity/proto3-suite
+  commit: HEAD
```

Comment:
That is just asking to break. Let’s fix the commits that we use as well.

Reply:
fixed as you suggested